data/developerhub/applications.json

```diff
@@ -0,0 +1,70 @@
+{
+  "applications": [
+    {
+      "title": "Serverless Container-based APIs with Amazon ECS & API Gateway",
+      "description": "Deploy a Full-Stack Serverless Web application, and deploy it with Terraform & CloudFormation on LocalStack",
+      "url": "https://github.com/localstack/serverless-api-ecs-apigateway-sample",
+      "teaser": "https://raw.githubusercontent.com/localstack/serverless-api-ecs-apigateway-sample/main/images/serverless-container-api.png",
+      "services": ["ddb", "agw", "iam", "cgn", "ecs"],
+      "deployment":  ["terraform", "awscli","cloudformation"],
+      "platform": ["javascript"],
+      "tags": ["serverless-containers", "security", "identity", "compliance"],
+      "pro": true
+    },
+    {
+      "title": "Full-Stack application with AWS Lambda, DynamoDB & S3 for shipment validation",
+      "description": "Configure a CRUD web application for shipment validation & listing, and deploy it with Terraform on LocalStack",
+      "url": "https://github.com/localstack/shipment-list-demo",
+      "teaser": "https://raw.githubusercontent.com/localstack/shipment-list-demo/master/app_diagram.png",
+      "services": ["s3", "ddb", "lmb", "iam"],
+      "deployment":  ["terraform"],
+      "platform": ["java", "react"],
+      "tags": ["spring boot", "lambda-trigger"],
```

Comment:
nit: I noticed that some of the tags use hyphens, whereas others don't. Should we maybe standardize on one common format?